R/graph_layout_visualization.R

```diff
@@ -535,7 +535,7 @@ Plot3DGraph <- function (
   cell_id,
   marker = NULL,
   assay = NULL,
-  layout_method = c("pmds", "wpmds", "fr", "kk", "drl"),
+  layout_method = "pmds",
```

Comment:
Is there a reason to remove the vector of choices from before?

Reply:
Yes! These are the default options in `ComputeLayout`, but technically you could provide any custom method to `ComputeLayout` and give it any name you want. `Plot3DGraph` should be able to plot any layout as long as it is 3D.

Also, when running Pixelator with pre-computed layouts activated, the layout are named like this:
- "pmds" -> 2D layout
- "pmds_3d" -> 3D layout
etc.

In pixelatorR, 2D and 3D layouts have the same name. You just need decide if you want to compute with dim=2 or dim=3, but you cannot do both unless you set the `layout_name` manually. We might change this in the future, but right now we need a way to enable plotting of "pmds_3d" created with Pixelator. By removing these choices, you can simply set `layout_method = "pmds_3d"` if you have loaded the pre-computed layouts.